Java/core/src/main/java/com/amazon/randomcutforest/store/PointStore.java

```diff
@@ -211,7 +208,8 @@ public int add(double[] point, long sequenceNum) {
 
         double[] tempPoint = point;
         if (internalShinglingEnabled) {
-            tempPoint = changeShingleInPlace(internalShingle, point);
+            // rotation is supported via the output and inoput is unchanged
+            tempPoint = changeShingleInPlace(internalShingle, point, false);
```

Comment:
`changeShingleInPlace` is not a very descriptive name. change how?

Reply:
renaming to constructShingleInPlace. The goal was to avoid making copies.